packages/context/src/interceptor.ts

```diff
@@ -60,7 +60,9 @@ export class InvocationContext extends Context {
    * @param args - An array of arguments
    */
   constructor(
-    parent: Context,
+    // Make `parent` public so that the interceptor can add bindings to
+    // the request context, for example, tracing id
```

Comment:
Is this relying on the fact that interceptor's context has the request context as the parent? I find that rather brittle, what if we introduce another context in between per-request and interceptor/invocation? 

IMO, interceptors wishing to contribute bindings to per-request context should receive that context via DI. See
https://github.com/strongloop/loopback-next/blob/6caceda5a7f8f83725b4fe79447df4d4b9258d1e/packages/rest/src/keys.ts#L229-L231

Reply:
Your proposal is better. I can revert the change.